plugins/inputs/timex/README.md

```diff
@@ -0,0 +1,74 @@
+# Timex Input Plugin
+
+This plugin gathers metrics on system time.
```

Comment:
Please add a reference (i.e. a link to further documentation) on the mechanism used to gather the time.